api/credential_request.go

```diff
@@ -0,0 +1,86 @@
+package api
+
+import (
+	"github.com/infrahq/infra/uid"
+)
+
+type CreateCredentialRequest struct {
+	Destination string `form:"destination"`
+}
+
+type ListCredentialRequest struct {
+	Destination     string `form:"destination"`
+	LastUpdateIndex int64  `form:"lastUpdateIndex"`
+}
+
+type ListCredentialRequestResponse struct {
+	Items          []CredentialRequest `json:"items"`
+	MaxUpdateIndex int64               `json:"maxUpdateIndex"`
+}
+
+// CredentialRequest is the database CredentialRequest object, not a request/response struct
+type CredentialRequest struct {
```

Comment:
This is going to store credentials to access the destination where the connector is running?

Reply:
no, just on the api.